.goreleaser.yml

```diff
@@ -17,14 +17,14 @@ builds:
   ldflags:
   - -w -X github.com/tektoncd/cli/pkg/cmd/version.clientVersion={{.Version}}
 archives:
-- name_template: "{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
-  replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
-    arm64: aarch64
+- name_template: >-
+    {{- .Binary }}_
+    {{- .Version }}_
+    {{- title .Os }}_
```

Comment:
do we need to do title here ?

Reply:
since Os value can be one of `darwin`, `linux` or `windows` and we had a replacements for the same
```
darwin: Darwin
linux: Linux
windows: Windows
``` 
so in order to replicate the same, added `title .Os`, do let me know if we are fine with having them in lower case